src/test/java/RegistrationTest.java

```diff
@@ -0,0 +1,20 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RegistrationTest extends BaseTest {
+    @Test
+    public void registrationNavigation() {
+        openUrl();
+        WebElement registrationButton = driver.findElement(By.cssSelector("#hel"));
+        registrationButton.click();
+        WebElement registerButton = driver.findElement(By.cssSelector("#button"));
+        Assert.assertTrue(registerButton.isDisplayed());
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._